src/Popover.vue

```diff
@@ -112,15 +112,17 @@ export default {
           this.visible = true
 
           this.$nextTick(() => {
-            let position = this
-              .getDropdownPosition(target, this.$refs.dropdown, direction)
-
-            this.position = {
-              left: `${position.left}px`,
-              top: `${position.top}px`
-            }
-
-            this.$emit('show', { ...event, position })
+            this.$emit('show', event)
+            
+            setTimeout(()=> {
```

Comment:
I think its better to use `nextTick`, see line 114

Reply:
like this.
```js
this.$nextTick(() => {
    this.$emit('show', { ...event, position }) 
    this.$nextTick(()=>{
            let position = this
              .getDropdownPosition(target, this.$refs.dropdown, direction)
            this.position = {
              left: `${position.left}px`,
              top: `${position.top}px`
            }
    })
})
```